meshtastic/tests/test_mesh_interface_behavioral_compat.py

```
@@ -0,0 +1,1851 @@
+            ) as mock_send:
+                mock_packet = MagicMock()
+                mock_packet.id = 10000 + hash(telemetry_type) % 10000
+                mock_send.return_value = mock_packet
+
+                try:
+                    iface.sendTelemetry(
+                        destinationId=BROADCAST_ADDR,
+                        telemetryType=telemetry_type,
+                        wantResponse=False,
+                    )
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)